styles/globals.css

```diff
@@ -1,139 +1,143 @@
-    justify-content: center;
+  width: 100%;
+  padding: 8px 42px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .UsecaseWrapper_wrapper {
-    padding: 0 16px;
-    max-width: 600px;
-    flex: 1 1;
+  padding: 0 16px;
+  max-width: 600px;
+  flex: 1 1;
 }
 
 .UsecaseWrapper_wrapper.full {
-    max-width: initial;
+  max-width: initial;
 }
 
 .UsecaseWrapper_title {
-    width: 100%;
-    font-weight: 500;
-    font-size: 32px;
+  width: 100%;
+  font-weight: 500;
+  font-size: 32px;
 }
 
 .UsecaseWrapper_helper {
-    margin-bottom: 1rem;
-    text-align: left;
-    opacity: 0.72;
-    font-size: 16px;
+  margin-bottom: 1rem;
+  text-align: left;
+  opacity: 0.72;
+  font-size: 16px;
 }
 
-
-
 .UsecaseWrapper_notes {
-    padding-left: 16px;
-    font-size: 14px;
-    font-weight: 400;
-    margin-top: 1rem;
-    margin-bottom: 0.75rem;
-    text-align: left;
+  font-size: 14px;
+  font-weight: 400;
+  margin-top: 1rem;
+  margin-bottom: 0.75rem;
+  text-align: left;
 }
 
 .UsecaseWrapper_notes li {
-    margin-bottom: 12px;
+  margin-bottom: 12px;
 }
 
 .UsecaseWrapper_alert {
-    margin-top: 36px;
+  margin-top: 36px;
 }
 
 .UsecaseWrapper_alert {
-    margin-top: 2.5rem;
-    margin-bottom: 2.5rem;
+  margin-top: 2.5rem;
+  margin-bottom: 2.5rem;
 }
 
 .ActionWrapper_container.full {
-    box-shadow: none !important;
-    padding-left: 0;
-    padding-right: 0;
+  box-shadow: none !important;
+  padding-left: 0;
+  padding-right: 0;
 }
 
 .ActionWrapper_container {
-    padding: 42px 36px;
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    flex-direction: column;
-    position: relative;
-    overflow: hidden;
-    box-shadow: 0 8px 10px rgb(0 0 0 / 5%), 0 16px 24px rgb(0 0 0 / 8%), 0 6px 30px rgb(0 0 0 / 5%) !important;
+  padding: 42px 36px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: column;
+  position: relative;
+  overflow: hidden;
+  box-shadow: 0 8px 10px rgb(0 0 0 / 5%), 0 16px 24px rgb(0 0 0 / 8%), 0 6px 30px rgb(0 0 0 / 5%) !important;
 }
 
 .Form_container {
-    width: 100%;
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    flex-direction: column;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: column;
 }
 
 .Form_container > .Form_button,
 .Form_container > .Form_button {
-    margin-top: 24px;
+  margin-top: 24px;
 }
 
 .UserInput_label {
-    margin-bottom: 4px !important;
-    font-size: 14px !important;
+  margin-bottom: 4px !important;
+  font-size: 14px !important;
 }
 
 /* Base */
 
 a {
-    color: var(--primary);
-    text-decoration: none;
+  color: var(--primary);
+  text-decoration: none;
 }
 
 code {
-    font-size: 1rem;
-    color: #d63384;
-    word-wrap: break-word;
+  font-size: 1rem;
+  color: #d63384;
+  word-wrap: break-word;
+}
+
+/* Utility classses */
+.nowrap {
```

Comment:
Do we need utility classes? I feel that it would be just easier to use `sx` prop or <Box /> component

Reply:
In this case, I am trying to apply just one CSS rule to a plain HTML element, not an MUI component. Making a particular class just for that felt like overkill and it seemed like the class would be helpful in the future, but I am not super familiar with MUI styling as you might have noticed :D 

Is there a better MUI way to do it? Or should I make a CSS class for the one element? Or use inline styles? Thanks! :)

Reply:
Yep! :)